src/Components/Bookslice.js

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const bookSlice = createSlice({
```

Comment:
- [ ] Good job so far 😃, however, as it is [required](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions_v2.md#:~:text=Structure%20your%20application,%7C%20index.js), the structure of the files should follow the `ducks pattern`: 
````
/src
    |---/components
    |---/redux
        |--- /books
                | books.js
        |--- /categories
                | categories.js
        | store.js
    | index.js
````
Make sure your `Categoryslice.js` and `booksSlice.js` are in the `redux` folder, It'll be something like this:

````
redux/books/booksSlice.js
redux/categories/categoriesSlice.js

````

Kindly, fix it 👍